content/docs/iac/concepts/projects/_index.md

```diff
@@ -8,7 +8,7 @@ menu:
     iac:
         name: Projects
         parent: iac-concepts
-        weight: 20
+        weight: 1
```

Comment:
Curious why this change?  Is it intended in this PR?

Reply:
No... not sure how that changed